soccer/gameplay/tactics/wall.py

```diff
@@ -0,0 +1,65 @@
+import composite_behavior
+import behavior
+import constants
+import robocup
+import main
+import enum
+import math
+import skills.move
+
+# This tactic builds a wall a certain distance from point A, blocking point B.
+class Wall(composite_behavior.CompositeBehavior):
+    class State(enum.Enum):
+        defense_wall = 1
+
+    def __init__(self,
+                 num_defenders = 3,                         # number of defenders we're making the wall with (default 3)
+                 curvature = 0,                             # curvature (in radians) of the wall 0 - pi/2
+                 mark_point = None,                         # what point we are defending against (default is ball)
+                 defender_point = robocup.Point(0, 0),      # what point we are defending (default is goal)
+                 defender_spacing = 3,                      # number of robot radii between the centers of the defenders in the wall
```

Comment:
This feels super high in terms of spacing. We'll have to test it a little more, but it should probably be lower

Reply:
Just looking at the simulator, the spacing looked fine, but I agree some onfield testing would probably be good to verify that and would be useful in general (for path planning as well).